lib/pmon.c

```diff
@@ -348,6 +348,60 @@ void switchtec_bwcntr_sub(struct switchtec_bwcntr_res *new,
 	new->ingress.comp -= old->ingress.comp;
 }
 
+/**
+ * @brief Set bandwidth type for a number of ports
+ * @param[in]  dev		Switchtec device handle
+ * @param[in]  nr_ports		Number of ports to set
+ * @param[in]  phys_port_ids	The physical ids for each port to set
+ * @param[in]  bw_type		0 if raw bandwidth, 1 if payload bandwidth
```

Comment:
Please use an enum for bandwidth type so the code will be more descriptive than 0 and 1.